src/gwkokab/inference/poissonlikelihood.py

```
@@ -24,6 +24,7 @@ def poisson_likelihood(
     data: List[np.ndarray],
     log_ref_priors: List[np.ndarray],
     ERate_fn: Callable[[Distribution], Array],
```

Pylint (3.3.7)

[convention] Argument name "ERate_fn" doesn't conform to snake_case naming style

(C0103)